config/environments/development.rb

```diff
@@ -30,54 +32,36 @@
     config.cache_store = :null_store
   end
 
+  # Don't care if the mailer can't send.
+  config.action_mailer.raise_delivery_errors = false
+
+  config.action_mailer.perform_caching = false
+
   # Print deprecation notices to the Rails logger.
   config.active_support.deprecation = :log
 
+  # Raise exceptions for disallowed deprecations.
+  config.active_support.disallowed_deprecation = :raise
+
+  # Tell Active Support which deprecation messages to disallow.
+  config.active_support.disallowed_deprecation_warnings = []
+
   # Raise an error on page load if there are pending migrations.
   config.active_record.migration_error = :page_load
 
   # Highlight code that triggered database queries in logs.
   config.active_record.verbose_query_logs = true
 
-  # Debug mode disables concatenation and preprocessing of assets.
-  # This option may cause significant delays in view rendering with a large
-  # number of complex assets.
-  # config.assets.debug = true
-
-  # Suppress logger output for asset requests.
-  # config.assets.quiet = true
-
   # Raises error for missing translations.
-  # config.action_view.raise_on_missing_translations = true
+  config.i18n.raise_on_missing_translations = true
+
+  # Annotate rendered view with file names.
+  # config.action_view.annotate_rendered_view_with_filenames = true
 
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 
-  # have puma server output to stdout (when started via `foreman start`)
-  config.logger = Logger.new($stdout)
-
-  # see https://github.com/shadabahmed/logstasher
-  config.logstasher.enabled = true
-  config.logstasher.log_level = Logger::INFO
-  config.logstasher.controller_enabled = true
-  config.logstasher.mailer_enabled = false
-  config.logstasher.record_enabled = false
-  config.logstasher.view_enabled = false
-  config.logstasher.job_enabled = true
-  config.logstasher.suppress_app_log = false
-  config.logstasher.source = 'laa-court-data-ui-development'
-  config.logstasher.backtrace = true
-  config.logstasher.logger_path = 'log/logstasher_development.log'
```

Comment:
Should these lines have been removed? Looks like they are config for logstasher and not rails related.

Reply:
I've reinstated this as it seems it was removed accidentally